src/github.com/getlantern/flashlight/client/bootstrap.go

```diff
@@ -147,9 +146,11 @@ func bootstrapPath(fileName string) (string, error) {
 			yamldir = dir + "/../Resources"
 		}
 	} else if runtime.GOOS == "linux" {
-		yamldir = dir
+		yamldir = dir + "/../"
```

Comment:
On linux, executable is started from `~/.lantern/bin`, and the config file is placed at `~/.lantern`